[2023-04-17 17:36:22,362] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-17T17:36:20.292137+00:00 [queued]>
[2023-04-17 17:36:22,371] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-17T17:36:20.292137+00:00 [queued]>
[2023-04-17 17:36:22,373] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-17 17:36:22,373] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-17 17:36:22,374] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-17 17:36:22,387] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_cv_lac_data> on 2023-04-17 17:36:20.292137+00:00
[2023-04-17 17:36:22,449] {standard_task_runner.py:52} INFO - Started process 456 to run task
[2023-04-17 17:36:22,491] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'get_cv_lac_data', 'manual__2023-04-17T17:36:20.292137+00:00', '--job-id', '208', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmpq0qzdqgr', '--error-file', '/tmp/tmpk2c9x02y']
[2023-04-17 17:36:22,494] {standard_task_runner.py:80} INFO - Job 208: Subtask get_cv_lac_data
[2023-04-17 17:36:22,880] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-17T17:36:20.292137+00:00 [running]> on host 6071715564b2
[2023-04-17 17:36:23,341] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=get_cv_lac_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-17T17:36:20.292137+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-17T17:36:20.292137+00:00
[2023-04-17 17:36:23,373] {logging_mixin.py:115} INFO - Getting CV_LAC Data
[2023-04-17 17:36:23,377] {python.py:173} INFO - Done. Returned value was: None
[2023-04-17 17:36:23,397] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=get_cv_lac_data, execution_date=20230417T173620, start_date=20230417T173622, end_date=20230417T173623
[2023-04-17 17:36:23,447] {local_task_job.py:156} INFO - Task exited with return code 0
